packages/fuel-core/VERSION

```diff
@@ -1 +1 @@
-0.19.0
+0.19.1
```

Comment:
Do we need this version bump for the changes in this PR to work or is it unrelated?

Reply:
Is not completely related but it has a fix for GTF message data.